demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -1201,3 +1213,206 @@ def test_IsEntityNameContainExcludedWordValidator(
     assert len(results) == expected_number_of_failures
     if results:
         assert results[0].message == expected_error_message
+
+
+@pytest.mark.parametrize(
+    "content_items, expected_number_of_failures, expected_field_error_messages",
+    [
+        pytest.param(
+            [
+                create_incident_field_object(),
+                create_widget_object(),
+                create_wizard_object(),
+                create_report_object(),
+                create_xsiam_report_object(),
+                create_script_object(),
+                create_dashboard_object(),
+                create_incident_type_object(),
+                create_generic_module_object(),
+                create_generic_type_object(),
+                create_incoming_mapper_object(),
+                create_outgoing_mapper_object(),
+                create_generic_definition_object(),
+                create_classifier_object(),
+                create_xsiam_dashboard_object(),
+                create_job_object(),
+                create_list_object(),
+                create_parsing_rule_object(),
+                create_playbook_object(),
+                create_generic_field_object(),
+                create_correlation_rule_object(),
+                create_assets_modeling_rule_object(),
+                create_layout_object(),
+            ],
+            0,
+            [],
+            id="All the content items, no failure expected",
+        ),
+        pytest.param(
+            [
+                create_classifier_object(
+                    paths=["name", "id"],
+                    values=[FIELD_WITH_WHITESPACES, FIELD_WITH_WHITESPACES],
+                ),
+                create_integration_object(
+                    paths=["name", "commonfields.id"],
+                    values=[FIELD_WITH_WHITESPACES, FIELD_WITH_WHITESPACES],
+                ),
+            ],
+            2,
+            ["object_id, name" for _ in range(2)],
+            id="classifier and integration with trailing spaces",
+        ),
+        pytest.param(
+            [
+                create_indicator_field_object(
+                    paths=["name"], values=[FIELD_WITH_WHITESPACES]
+                ),
+                create_wizard_object({"name": FIELD_WITH_WHITESPACES}),
```

Comment:
can we use this dictionary-style instead of passing `paths=..., values=...` in all cases? It's preferred (less chance for index errors, more readable)